src/main/java/core/basesyntax/Printer.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Printer {

```

Comment:
If you have method `draw()` maybe does it better to have class name as `Drawer`?